README.md

```diff
@@ -76,6 +76,9 @@ androidDisplayNotificationProgress | `boolean` | (Android only) Used to set if p
 androidNotificationTitle | `string` | (Android only) Used to set the title shown in the Android notifications center.
 androidAutoDeleteAfterUpload | `boolean` | (Android only) Used to set if files should be deleted automatically after upload.
 androidMaxRetries | `number` | (Android only) Used to set the maximum retry count. The default retry count is 0. https://github.com/gotev/android-upload-service/wiki/Recipes#backoff
+androidAutoClearNotification | `boolean` | (Android only) Used to set if notifications should be cleared automatically upon upload completion. Default is false.
+androidRingToneEnabled | `boolean` | (Android only) Used to set if a ringtone should be played upon upload completion. Default is true.
```

Comment:
If this and `androidAutoClearNotification` are both set to true there will not be an ringtone since the notification is automatically cleared. Can you add this as a note here.